src/components/services/PiHole.vue

```diff
@@ -49,7 +52,8 @@ export default {
   },
   methods: {
     fetchStatus: async function () {
-      const result = await this.fetch("/api.php").catch((e) => console.log(e));
+      const result = await this.fetch("`/api.php${this.apiQuery}`")
```

Comment:
Something like this should to the job
```suggestion
      const authQueryParams = this.item.apikey ? `?summaryRaw&auth=${this.item.apikey}` : "";
      const result = await this.fetch(`/api.php${authQueryParams}`)
```